### PR TITLE
[ISSUE #4150] Fix spelling problem on docs/en

### PR DESCRIPTION
**Pull request by gogodjzhu**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

Fix some spelling problem on docs.

## Brief changelog

Almost all docs on `docs/en` directory.

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [ ] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by dugenkui03 — APPROVED** [review]

**Comment by dugenkui03** [comment]
提交包括其他人commit

File: docs/en/Example_OpenMessaging.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # OpenMessaging Example
-[OpenMessaging](https://openmessaging.github.io/), which includes the establishment of industry guidelines and messaging, streaming specifications to provide a common framework for finance, ecommerce, IoT and big-data area. The design principles are the cloud-oriented, simplicity, flexibility, and language independent in distributed heterogeneous environments. Conformance to these specifications will make it possible to develop a heterogeneous messaging applications across all major platforms and operating systems.
+[OpenMessaging](https://openmessaging.github.io/), which includes the establishment of industry guidelines and messaging, streaming specifications to provide a common framework for finance, commerce, IoT and big-data area. The design principles are the cloud-oriented, simplicity, flexibility, and language independent in distributed heterogeneous environments. Conformance to these specifications will make it possible to develop a heterogeneous messaging applications across all major platforms and operating systems.
```
Comment (vongosling) [inline docs/en/Example_OpenMessaging.md:2]: e-commerce would be better

**Review by vongosling — CHANGES_REQUESTED** [review]

**Comment by vongosling** [comment]
@gogodjzhu @dugenkui03 very much appreciate your careful revision :-) I hope to see more of you in the community.

File: docs/en/Troubleshoopting.md [hunk 2]
```diff
@@ -73,4 +73,4 @@ SendResult [sendStatus=SEND_OK, msgId=0A42333A0DC818B4AAC246C290FD0000, offsetMs
 ```
 
 - msgId，for the client, the msgId is generated by the producer instance. Specifically, the method `MessageClientIDSetter.createUniqIDBuffer()` is called to generate a unique Id.
-- offsetMsgId, offsetMsgId is generated by the Broker server when writing a message ( string concating "IP address + port" and "CommitLog's physical offset address"), and offsetMsgId is the messageId used to query in the RocketMQ console.
+- offsetMsgId, offsetMsgId is generated by the Broker server when writing a message ( string concat with "IP address + port" and "CommitLog's physical offset address"), and offsetMsgId is the messageId used to query in the RocketMQ console.
```
Comment (yuz10) [inline docs/en/Troubleshoopting.md:76]: IMO, better use  ` string consisting of "IP address + port" and "CommitLog's physical offset address"`

**Comment by gogodjzhu** [comment]
> 提交包括其他人commit

**Comment by gogodjzhu** [comment]
Fresher in the town :) Thanks for the suggestions and changes has been commited. @dugenkui03 @vongosling @yuz10

**Review by yuz10 — APPROVED** [review]

**Comment by vongosling** [comment]
The last step. It would be helpful to create a related issue and associate with it, just like others do. It is also our basic community requirements :-)

**Comment by gogodjzhu** [comment]
this pr is related to #4141 

@vongosling Next time will be issue first and pr follow.

**Review by panzhi33 — APPROVED** [review]

**Pull request merged**